common/src/main/java/com/wynntils/handlers/command/CommandHandler.java

```diff
@@ -14,9 +14,11 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class CommandHandler extends Handler {
+    private static final int TICKS_PER_EXECUTE = 7;
+    private static final int NPC_DIALOGUE_WAIT_TICKS = 40;
```

Comment:
This feels a bit arbitrary. Is it to wait for the player to finish the dialogue interaction, or for wynn to send all parts of the current dialogue screen? I'm not actually sure I understand how this is a fix, or what it is supposed to fix.

Reply:
It's supposed to wait for us to fully finish a dialogue.